src/traits/interface.rs

```diff
@@ -8,17 +8,21 @@ pub unsafe trait Interface: Sized + Abi {
     type Vtable;
     const IID: Guid;
 
+    /// Returns the vtable for the current interface.
     unsafe fn vtable(&self) -> &Self::Vtable {
```

Comment:
We should document when this methods are safe to call. Typically one would add a `# Safety` section to the docs which outlines this.

Reply:
https://github.com/microsoft/winrt-rs/issues/359